internals/models.py

```diff
@@ -1236,6 +1272,7 @@ def put(self, notify=True, **kwargs):
 
   # General info.
   category = ndb.IntegerProperty(required=True)
+  creator = ndb.StringProperty()
```

Comment:
We don't need a new field for creator because we already have created_by.

Reply:
I thought as much, but then I saw this comment:
https://github.com/GoogleChrome/chromium-dashboard/blob/1b142660f5abbeae171b6146913022e7e926339a/internals/models.py#L1367-L1370

I did try and query `created_by` anyway, but I was not able to get it to work properly. After reading some common problems, it seemed like other devs recommended keeping a separate `created` field if it needs to be referenced, as `created_by` comes with additional baggage being a `UserProperty`.
However, there might be a way to use it - this is just the solution I came up with.